server/models/transactions.js

```diff
@@ -1,10 +1,29 @@
 // imports goes here
-class Transaction {
+import uuidv4 from 'uuid/v4';
```

Comment:
Instead of adding another library for generating UUIDs, I think that we should be using Postgresql UUID extension. With the extension the uuids are auto-generated.  Follow this [link](https://stackoverflow.com/questions/12505158/generating-a-uuid-in-postgres-for-insert-statement) to see how to do it or ask Eric to help you out with that.